scripts/west_commands/build.py

```diff
@@ -139,6 +139,10 @@ def do_add_parser(self, parser_adder):
                             help='Force CMake to run')
         parser.add_argument('--just-cmake', action='store_true',
                             help="Just run CMake; don't build. Implies -c.")
+        parser.add_argument('-n', '--just-print', '--dry-run', '--recon',
```

Comment:
do we really need that many names? `-n`/`--dry-run` should be enough?

Reply:
Those are the names Make uses. I'm making sure people get all of them.